tests/javascript_newsletter_editor/mochaTestHelper.js

```diff
@@ -15,16 +15,17 @@ global.sinon = sinon;
 global.URL = URL;
 
 if (!global.document || !global.window) {
-  global.document = jsdom('<html><head><script></script></head><body></body></html>', {
-    url: 'http://example.com',
-  }, {
-    FetchExternalResources: ['script'],
```

Comment:
jsdom doesn't accept a third parameter anymore and those configurations were replaced with `runScripts` and `resources`.